firefox_puppeteer/api/utils.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class Utils(BaseLib):
+    """Low-level access to utility actions."""
+
+    def remove_perms(self, host, permission):
+        """Remove permission for host.
+
+        :param host: Web host, e.g., www.itisatrap.org
+        :param permission: Permission type, e.g., 'safe-browsing'
```

Comment:
We might want to improve the documentation later for that part to make it easier for testers to use it. No-one knows what to use e.g. as permission type.